src/blocks/plugin-card/edit.js

```diff
@@ -42,7 +44,7 @@ const Edit = ({
 			<Disabled>
 				<ServerSideRender
 					block="themeisle-blocks/plugin-cards"
-					attributes={ { ...attributes } }
+					attributes={ { ...omit( attributes, [ 'className' ]) } }
```

Comment:
While this will fix the error, it won't fix the issue. className attribute won't be used in the rendered block in the backend which is an issue.